lib/chooseTeam.js

```diff
@@ -0,0 +1,43 @@
+const rl = require("./rl");
+const logGrid = require("./logGrid");
+const checkWin = require("./checkWin");
+
+const getTeamChoice = () => {
```

Comment:
* rename this to `promptForTeamChoice()`
* make new function called getTeamChoice which has a loop (or is recursive) and calls `promptForTeamChoice()` until the result passes `isValidTeamChoice()` (at which point, return the valid choice)